src/mimemail.erl

```diff
@@ -2429,8 +2398,10 @@ rfc2047_decode_test_() ->
             ?assertEqual(
                 <<"ab">>,
                 decode_header(
-                    <<"=?ISO-8859-1?Q?a?=\n"
-                    "		=?ISO-8859-1?Q?b?=">>,
+                    <<
+                        "=?ISO-8859-1?Q?a?=\n"
+                        "		=?ISO-8859-1?Q?b?="
+                    >>,
```

Comment:
This was not part of my original PR, it got reformatted with the rest when I ran `rebar3 fmt`.